codice/ddf-ui
# [4.2.x] Remove frontend usage of federation, updates query to only use user defaults if flag is passed

- Updates query to by default not use the user specified settings.
 - Removes code paths that are no longer needed.

Testing:  
 - Update your user settings to have a selected set of sources (such as slow).
 - Verify creating a new search will utilize those sources.
 - Verify that when visiting the id page of a metacard, the query that goes out does not use the users setting for sources (set it to something that the metacard isn't in, or just look at the network).
